pkg/operator/ceph/cluster/watcher.go

```diff
@@ -91,9 +91,6 @@ func (c *clientCluster) onK8sNode(ctx context.Context, object runtime.Object, op
 	cluster := c.getCephCluster()
 
 	// Continue reconcile in case of failure too since we don't want to block other node reconcile
-	if err := c.handleNodeFailure(ctx, cluster, node, opNamespace); err != nil {
```

Comment:
instead of removing can we add a 
if (1!=1) false test
To avoid `handleNodeFailure` func not called issues in some lint

Reply:
Instead of that condition, I'd suggest commenting the lines so we can see where to restore them later

Reply:
@travisn i commented the code, PTAL